cli/lib/kontena/cli/stacks/logs_command.rb

```diff
@@ -9,13 +9,20 @@ class LogsCommand < Kontena::Command
     banner "Shows logs from services in a stack"
 
     parameter "NAME", "Stack name"
+    parameter "[SERVICE] ...", "Service names"
 
     requires_current_master
     requires_current_master_token
 
     def execute
-      show_logs("stacks/#{current_grid}/#{name}/container_logs") do |log|
-        show_log(log)
+      if service_list.empty?
+        show_logs("stacks/#{current_grid}/#{name}/container_logs") do |log|
+          show_log(log)
+        end
+      else
+        show_logs("grids/#{current_grid}/container_logs", services: service_list.map {|s| [current_grid, name, s].join('/')}.join(',')) do |log|
```

Comment:
The `grids `API params probably should not need to include the actual grid name, so just `?service=stack/service`?